en/08_Changelogs/6.0.0.md

```diff
@@ -4,61 +4,63 @@ title: 6.0.0 (unreleased)
 
 # 6.0.0 (unreleased)
 
+Silverstripe CMS 6.0.0 is a major release. That means it contains many breaking changes and dependency upgrades. We tried our best to minimise upgrades woes, but you should allocate additional time to account for regression testing.
+
+This changelog provides a list of changes between Silverstripe CMS 5.4 and 6.0.
+
 ## Overview
 
 - [Change to commercially supported modules](#changes-to-support)
   - [Modules losing commercial support](#modules-losing-support)
+  - [`silverstripe/campaign-admin` no longer part of `silverstripe/recipe-cms`](#campaign-admin)
   - [New supported modules](#new-supported-modules)
+  - [New default front-end theme](#theme)
+  - [TinyMCE is in a separate module now](#tinymce)
```

Comment:
Moved things that are directly related to changes in supported modules into this section